### PR TITLE
put train/val mini times into Parameters.py

### PR DESCRIPTION
**Pull request by karlzipser**
This pull request reflects minor changes. It is mainly intended as a test of the collaborative system.

### CODE REVIEW

File: Train.py [hunk 1]
```diff
@@ -76,14 +76,12 @@
...
                     batch.display()
-
                     plt.figure('loss')
                     plt.clf()  # clears figure
                     loss_record['train'].plot('b')  # plot with blue color
                     loss_record['val'].plot('r')  # plot with red color
                     print_timer.reset()
 
-            batch = Batch.Batch(net)  # Reinitialiize batch
+            batch = Batch.Batch(net)  # Reinitialiize batch. ***Why is this necessary?***
```
Comment (raunakdoesdev) [inline Train.py:87]: This line serves to replace your batch.clear() in the previous version (pytorch2). I'll look into what that did, but I changed it from .clear to a re-initialization since it essentially does the same thing.

**Comment by raunakdoesdev** [comment]
Also, what does the mini word mean in "mini-train-time" and "mini-val-time"?

**Comment by karlzipser** [comment]
I train for 30 minutes and validate for 3 minutes. If I call it
'train-time' it sounds like the entire length of training, so I used 'mini'
to indicate this 30 minute/3 minute period. Another name is fine. It is
very handy to monitor validation continually in this way.

---------------------------------------
Karl Zipser, Ph.D.
Assistant Researcher
Helen Wills Neuroscience Institute,
Berkeley DeepDrive Center & Redwood Center for Theoretical Neuroscience
UC Berkeley
561 Evans hall, MC 3198
Berkeley, CA 94720-3198
. .... ...

File: Train.py (hunk 1 above)
Comment (raunakdoesdev) [reply Train.py:87]: I took a look, essentially this serves to reinitialize the Tensors for camera_data, metadata, and labels. This is because in the forward pass function, we don't initialize these variables, and just start concatenating too them.

I think a better solution is to take this out of the main training loop, create a clear function in the Batch class and reference it at the beginning of the forward pass function. I will amend this pull request with the required changes, before merging into the main repository.

**Comment by raunakdoesdev** [comment]
Ok I made the changes and merged!
